### PR TITLE
doc(archiving): rename Bonita Platform => Runtime

### PR DESCRIPTION
**Pull request by educhastenier**
+ fix a few sentences.

### CODE REVIEW

File: md/maintenance-operation.md [hunk 1]
```diff
@@ -7,36 +7,35 @@ Archive tables contain the execution history of the Bonita Platform. Some of the
...
 
 ::: info
-
-**Note**: After a migration to the bonita 7.7 version, the data of the `archive_contract_data` table has been moved to the new `arch_contract_data_backup` table for migration purpose.
-In that case, If the `live-migration` tool was not executed (please see [Bonita migration steps]|migrate-from-an-earlier-version-of-bonita-bpm.md), you will also need to drop the table `arch_contract_data_backup`:
-
-   
+**Note**: After a migration to version 7.7 of Bonita Runtime, the data of the `archive_contract_data` table has been moved
+to a new table named `arch_contract_data_backup`, for migration purposes.
+In that case, if the `live-migration` tool was not executed (please see [Bonita migration steps]|migrate-from-an-earlier-version-of-bonita-bpm.md),
```
Comment (baptistemesta) [inline md/maintenance-operation.md]:
```suggestion
In that case, if the `live-migration` tool was not executed (please see [Bonita migration steps](migrate-from-an-earlier-version-of-bonita-bpm.md)),
```

Comment (educhastenier) [reply md/maintenance-operation.md]: Good catch

File: md/maintenance-operation.md [hunk 2]
```diff
@@ -7,36 +7,35 @@ Archive tables contain the execution history of the Bonita Platform. Some of the
...
-
-_This operation is destructive, data will be not usable for audit or analytic purpose. It is highly recommended to backup data before the purge._
+_This operation is destructive, data will be not usable for audit or analytics purposes. It is highly recommended to backup data
+before the purge._
 :::
 
-_Keep in mind that the contract data's will continue to be archived and the table will continue to be filled,  even after purge has been done. Hoverer, it's possible to deactivate the archiving of `contract data's`, please see ([Configurable Archive](configurable-archive.md).)_  
-   
+_Keep in mind that the contract data will continue to be archived and the table will continue to grow, even after the purge has been done.
+However, from version 7.10, it is possible to deactivate the archiving of `contract data`, please see ([Configurable Archive](configurable-archive.md).)_
```
Comment (rbioteau) [inline md/maintenance-operation.md:25]: You talk a bout 7.10 in 7.9 branch. Is it intended ?

Comment (educhastenier) [reply md/maintenance-operation.md:25]: To push customer to migrate, maybe? ...

**Pull request merged**